src/App.js

```diff
@@ -1,12 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
 import './App.css';
+//import {firstInputPolyfill, resetFirstInputPolyfill} from "web-vitals/dist/modules/lib/polyfills/firstInputPolyfill";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+    const [strawberryCounter, setStrawberryCounter] = useState(0); console.log(strawberryCounter)
+    const [bananaCounter, setBananaCounter] = useState(0); console.log(bananaCounter)
+    const [appleCounter, setAppleCounter] = useState(0); console.log(appleCounter)
+    const [kiwiCounter, setKiwiCounter] = useState(0); console.log( kiwiCounter)
+    //const [kiwiCounter] = useEffect(0); console.log(useEffect())
+//reset naar initial state useeffect reset totonClick={() =>useEffect(0)}
+
+    const {register, handleSubmit, watch, errors} = useForm({
+        mode: 'onChange'
+    });
+    const watchAnders = watch("frequency", "somethingother");
```

Comment:
Als je deze defaultvalue wilt hanteren moet je er ook voor zorgen dat de radiobutton by default aangetikt staat.

Reply:
In welke context?